packages/server/lib/browsers/electron.js

```diff
@@ -415,6 +422,12 @@ module.exports = {
     return webContents.session.clearCache()
   },
 
+  _getUserAgent (webContents) {
+    debug('getting user agent')
```

Comment:
Suggest including the user agent in the debug log.

Reply:
@mschile how is something like:

```
  _getUserAgent (webContents) {
    debug('getting user agent')

    const userAgent = webContents.session.getUserAgent()

    debug('found user agent: %s', userAgent)

    return userAgent
  }
```

Reply:
Not sure the first log is really needed. How about:
```js
  _getUserAgent (webContents) {
    const userAgent = webContents.session.getUserAgent()

    debug('getting user agent: %s', userAgent)

    return userAgent
  }
```

Reply:
my only concern would be if getting the user agent failed for some reason, which would provide absent in the logs. But I think that is rare and not sure when that would happen, so I'm good with just adding the log after retrieving the UA.

Reply:
added in [366e5ff](https://github.com/cypress-io/cypress/pull/22958/commits/366e5ff782e768589141cd753e2d1aec01656306)